### DISCUSSION TITLE
How to hide Create & Create Another button on modal?

### DISCUSSION THREAD

**Discussion by l-yk-l** [filamentphp/filament#3347] [Help] [answered]
I want to hide "Create & Create another" button on modal by my Relation Manager.
I know how to do this on the creation page, but how to do the same for the modal?
Thank you in advance!

**Comment by danharrin** [comment, marked as answer]
`CreateAction::make()->disableCreateAnother()`

**Reply by l-yk-l** [reply]
Thank you very much!
